pkg/xen-tools/patches-6c9639a72f0ca3a9430ef75f375877182281fdef/01-enable-gmp-patching.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/stubdom/Makefile b/stubdom/Makefile
+index 8cf7131c6a..c6a8051c85 100644
+--- a/stubdom/Makefile
++++ b/stubdom/Makefile
+@@ -178,7 +178,7 @@ gmp-$(XEN_TARGET_ARCH): gmp-$(GMP_VERSION).tar.bz2 $(NEWLIB_STAMPFILE)
+ 	tar xjf $<
+ 	rm $@ -rf || :
+ 	mv gmp-$(GMP_VERSION) $@
+-	#patch -d $@ -p0 < gmp.patch
++	patch -d $@ -p0 < gmp.patch
```

Comment:
This should be changed to patch if the file exist, and upstream this. Will do in another PR.